mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearRegressionWithSGD.scala

```diff
@@ -59,6 +59,8 @@ class StreamingLinearRegressionWithSGD private[mllib] (
 
   val algorithm = new LinearRegressionWithSGD(stepSize, numIterations, miniBatchFraction)
 
+  var model: Option[LinearRegressionModel] = None
```

Comment:
I wonder if it should be a method? if that worked, or didn't, it would be an additional clue.